static/app/views/replays/detail/replayMetaData.tsx

```diff
@@ -1,194 +0,0 @@
-import type {ReplayError, ReplayRecord} from 'sentry/views/replays/types';
-
-type Props = {
-  replayErrors: ReplayError[];
-  replayRecord: ReplayRecord | undefined;
-};
-
-function ReplayMetaData({replayRecord, replayErrors}: Props) {
-  const {pathname, query} = useLocation();
-  const organization = useOrganization();
-  const {projects} = useProjects();
-
-  const hasErrorTab = organization.features.includes('session-replay-errors-tab');
-
-  const errorsTabHref = hasErrorTab
-    ? {
-        pathname,
-        query: {
-          ...query,
-          t_main: 'errors',
-        },
-      }
-    : {
-        pathname,
-        query: {
-          ...query,
-          t_main: 'console',
-          f_c_logLevel: 'issue',
-          f_c_search: undefined,
-        },
-      };
-
-  const errorCountByProject = useMemo(
-    () =>
-      Object.entries(countBy(replayErrors, 'project.name'))
-        .map(([projectSlug, count]) => ({
-          project: projects.find(p => p.slug === projectSlug),
-          count,
-        }))
-        // sort to prioritize the replay errors first
-        .sort(a => (a.project?.id !== replayRecord?.project_id ? 1 : -1)),
-    [replayErrors, projects, replayRecord]
-  );
-
-  return (
-    <KeyMetrics>
-      <KeyMetricLabel>{t('OS')}</KeyMetricLabel>
-      <KeyMetricData>
-        <ContextIcon
-          name={replayRecord?.os.name ?? ''}
-          version={replayRecord?.os.version ?? undefined}
-        />
-      </KeyMetricData>
-
-      <KeyMetricLabel>{t('Browser')}</KeyMetricLabel>
-      <KeyMetricData>
-        <ContextIcon
-          name={replayRecord?.browser.name ?? ''}
-          version={replayRecord?.browser.version ?? undefined}
-        />
-      </KeyMetricData>
-
-      <KeyMetricLabel>{t('Start Time')}</KeyMetricLabel>
-      <KeyMetricData>
-        {replayRecord ? (
-          <Fragment>
-            <IconCalendar color="gray300" />
-            <TimeSince date={replayRecord.started_at} unitStyle="regular" />
-          </Fragment>
-        ) : (
-          <HeaderPlaceholder width="80px" height="16px" />
-        )}
-      </KeyMetricData>
-      <KeyMetricLabel>{t('Errors')}</KeyMetricLabel>
-      <KeyMetricData>
-        {replayRecord ? (
-          <StyledLink to={errorsTabHref}>
-            {errorCountByProject.length > 0 ? (
-              <Fragment>
-                {errorCountByProject.length < 3 ? (
-                  errorCountByProject.map(({project, count}, idx) => (
-                    <ErrorCount key={idx} countErrors={count} project={project} />
-                  ))
-                ) : (
-                  <StackedErrorCount errorCounts={errorCountByProject} />
-                )}
-              </Fragment>
-            ) : (
-              <ErrorCount countErrors={0} />
-            )}
-          </StyledLink>
-        ) : (
-          <HeaderPlaceholder width="80px" height="16px" />
-        )}
-      </KeyMetricData>
```

Comment:
This part in here got simpler, and all the deps that it needs are in other files now. 

There are a bunch of dependent code, which is why git doesn't consider this a 'moved' file.